shared/src/commonMain/kotlin/com/example/myapplication/shared/main/PreviewMainComponent.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.myapplication.shared.main
+
+object PreviewMainComponent : MainComponent {
+    override fun onShowWelcomeClicked() = Unit
```

Comment:
Could you please use `{}` instead of `= Unit`? I really don't like this code style. Sorry for not having static checks setup.

Reply:
Fixed.

Reply:
Doesn't look like fixed.

Reply:
Sorry some change got leaked 🙏